niv-updater

```diff
@@ -163,7 +163,7 @@ formatIssueLinks() {
 createPullRequestsOnUpdate() {
     echo 'Checking for updates'
     if [[ -z $INPUT_PULL_REQUEST_BASE ]]; then
-        INPUT_PULL_REQUEST_BASE="$GITHUB_REF"
+        INPUT_PULL_REQUEST_BASE="${GITHUB_REF#refs/heads/}"
```

Comment:
Per documentation from https://docs.github.com/en/actions/learn-github-actions/variables

> The fully-formed ref of the branch or tag that triggered the workflow run. For workflows triggered by push, this is the branch or tag ref that was pushed. For workflows triggered by pull_request, this is the pull request merge branch. For workflows triggered by release, this is the release tag created. For other triggers, this is the branch or tag ref that triggered the workflow run. This is only set if a branch or tag is available for the event type. The ref given is fully-formed, meaning that for branches the format is refs/heads/<branch_name>, for pull requests it is refs/pull/<pr_number>/merge, and for tags it is refs/tags/<tag_name>. For example, refs/heads/feature-branch-1.

Thus, I suggest expanding for the other reasonable use case. I don't know how this would work for pull_request type, so skipping that.

```suggestion
        INPUT_PULL_REQUEST_BASE="${GITHUB_REF#refs/heads/}"
        INPUT_PULL_REQUEST_BASE="${INPUT_PULL_REQUEST_BASE#refs/tags/}"
```